tests/cli/test_buildspec.py

```diff
@@ -146,23 +146,35 @@ def test_buildspec_find_invalid():
     cache = BuildspecCache(configuration=configuration)
 
     # testing buildtest buildspec find invalid. This will assert SystemExit exception raised by sys.exit
-    with pytest.raises(SystemExit):
+    try:
         cache.print_invalid_buildspecs(error=True)
+    except SystemExit:
+        assert True
 
-    with pytest.raises(SystemExit):
+    try:
         cache.print_invalid_buildspecs(error=False)
+    except SystemExit:
+        assert True
 
-    with pytest.raises(SystemExit):
+    try:
         cache.print_invalid_buildspecs(error=True, terse=True)
+    except SystemExit:
+        assert True
```

Comment:
Also i think a `assert True` is not necessary but since you need a statement in the except you may try to assert the exception type or run `continue`. The `assert` statement is useless

Reply:
I guess 'continue' keyword works only with loops. Instead adding 'pass' is fine ? 
`except SystemExit:
        pass`

Reply:
yea i think `pass` should be fine